CRM/Member/BAO/Query.php

```diff
@@ -473,6 +473,7 @@ public static function buildSearchForm(&$form) {
     ));
 
     $form->addElement('text', 'member_source', ts('Source'));
+    $form->addElement('text', 'membership_id', ts('Membership ID'));
```

Comment:
Done (although it seems that it still allows you to enter text, and the up/down arrows don't really add extra functionality imho)